tools/assets-migration-to-screenly-pro.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import traceback
+import sh
+
+import click
+import requests
+import time
+
+from requests.auth import HTTPBasicAuth
+
+HOME = os.getenv('HOME', '/home/pi')
+
+BASE_API_SCREENLY_URL = 'https://api.screenlyapp.com'
+ASSETS_SCREENLY_OSE_API = 'http://127.0.0.1:8080/api/v1.1/assets'
+
+PORT_NGROK = 4040
+PORT = 8000
+
+ngrok_process = None
+
+token = None
+ngrok_public_url = None
+
+
+################################
+# Suprocesses
+################################
+
+def start_http_ngrok_process(try_connection=100):
+    global ngrok_process
```

Comment:
It seems ngrok_process is not used outside of this func - is it that needed for it to be global?